docs/slurm/main.md

```diff
@@ -38,15 +38,19 @@ There are two main ways to run programs on RCC clusters:
 
 * Passive via a "batch job" 
 
-In an interactive session, you will load [software modules](../software/faq.md) and run your scripts in real-time, whereas when submitting batch jobs, you specify the software modules to be loaded and scripts to be run in advance. 
+It is important to note that in both ways you are essentially submitting a job to the SLURM scheduler. You have to wait for the requested resource (i.e. number of CPU cores, memory, and GPUs, if any) to be allocated to your session or job.
 
-Interactive jobs allow you to actively interact with the program running on compute node/s (e.g., execute cells in a Jupyter Notebook) in real-time. This is great for exploratory work or troubleshooting. An interactive job will persist until you disconnect from the compute node or until you reach the maximum requested time. 
+Interactive jobs allow you to actively interact with the program running on compute node(s) (e.g., executing cells in a Jupyter Notebook or analyzing output data using GUI applications) in real-time. This is great for exploratory work or troubleshooting. An interactive job will persist until you disconnect from the compute node or until you reach the maximum requested time. In an interactive session, you will load the necessary [software modules](../software/faq.md) and run your scripts in real-time.
```

Comment:
- "This is great" => "This comes in handy" or "This is helpful"?
- Do we really have to list all of the "or"? E.g., "An interactive job will persist until...", it may be disconnected when the system goes down as well...
- Do we need to say the "real-time"? It's possible to leave the session, and then return back to it later.

Reply:
fixed